### PR TITLE
VNC to flv and GIF deployment

### PR DESCRIPTION
**Pull request by vprusa**
This is testing branch for [automation](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/vprusa-miq-mw-ui-tests/) for: 
- sync properties files to make job cloneable - Main [job](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/) will probably fail for this PR, but that will be fixed once configuration will be moved to main job  
- https://trello.com/c/9mfq62OG/42-investigate-deploying-screenshots-of-failed-ui-test-to-prs

### CODE REVIEW

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by mattmahoneyrh** [comment]
should be using the logger, instead of "print", unless there is a reason.

File: setup.sh [hunk 1]
```diff
@@ -25,6 +25,18 @@ pip install pycurl
 
 pip install mgmtsystem
 
+# setup test recorder
+wget https://pypi.python.org/packages/1e/8e/40c71faa24e19dab555eeb25d6c07efbc503e98b0344f0b4c3131f59947f/vnc2flv-20100207.tar.gz
+tar -zxvf vnc2flv-20100207.tar.gz
```
Comment (mattmahoneyrh) [inline setup.sh]: Is there a python module that can be pip installed, instead of using a URL that will most likely change?

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

File: setup.sh (hunk 1 above)
Comment (vprusa) [inline setup.sh]: ah, forgot to change that, there is a way how to install it as one-line command, URL is still there..
pip install http://pypi.python.org/packages/source/v/vnc2flv/vnc2flv-20100207.tar.gz

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by vprusa** [comment]
test this please

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by vprusa** [comment]
test this please

**Comment by hawkularqe** [comment]
Build Status Message: Build Result Message<br> 
Test FAILed. Browser: Firefox

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by vprusa** [comment]
test this please

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [110](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/110/)
<br> No records available

**Pull request closed without merging**